src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -625,24 +624,18 @@ const formConfig = {
                 viewField: ApplicantField,
               },
               items: {
-                'ui:options': {
-                  updateSchema: formData => {
-                    return {
-                      title: context =>
-                        titleUI(
-                          `${applicantWording(formData, context)} information`,
-                          `Next we'll ask more questions about ${applicantWording(
-                            formData,
-                            context,
-                            false,
-                            false,
-                          )}. This includes social security number, mailing address, 
+                ...titleUI(
+                  ({ formData }) => `${applicantWording(formData)} information`,
+                  ({
+                    formData,
+                  }) => `Next we'll ask more questions about ${applicantWording(
+                    formData,
+                    undefined,
+                    false,
+                  )}. This includes social security number, mailing address, 
```

Comment:
Since we're no longer doing the custom personalized language based on list loop index, got rid of this and all other instances of `updateSchema` in the list loop.